libraries/Microsoft.Bot.Builder/Teams/TeamsActivityHandler.cs

```diff
@@ -79,10 +34,10 @@ public override async Task OnTurnAsync(ITurnContext turnContext, CancellationTok
                 throw new ArgumentException($"{nameof(turnContext)}.Activity must have non-null Type.");
             }
 
-            if (turnContext.Activity.ChannelId == Channels.Msteams)
```

Comment:
Should we be creating this object if the ChannelId is not Msteams?

Reply:
the real dependency on having a teamId so I softened the check